### DISCUSSION TITLE
MultiSelect: Add props ‘option’ for the ‘chip’ slot

### DISCUSSION THREAD

**Discussion by lakkvak** [primefaces/community#2974] [PrimeVue]
When `optionValue` is specified in multiselect, the `chip` slot will pass the value by the specified key, which is not always convenient. For example, when a completely different field is displayed in the slot itself and you have to search for the required option by the incoming value. That's why I would like to have a prop with an `option` in the slot itself, if any

**Comment by james598henry** [comment]
Hello,
To improve convenience when using [adpworkforcenow](https://www.iadpworkforcenow.com) the optionValue in a multi select component, you can use a prop that directly accesses the option within the slot itself. This allows you to display a different field while still working with the value efficiently. This method avoids the need to search for the required option by the incoming value.

Best Regards,
James Henry

**Reply by lakkvak** [reply]
![image](https://github.com/user-attachments/assets/2be20ad5-f73e-424f-b63e-0478ee88222a)

But there are no other props in the slot other than `value`
